.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - checkout
       - setup_remote_docker
       - restore_cache:
-          key: venv-{{ checksum "dev-requirements-py2.txt" }}-{{ checksum "dev-requirements-py3.txt" }}-{{ checksum ".circleci/build-requirements.txt" }}
+          key: venv-{{ checksum "dev-requirements-py2.txt" }}-{{ checksum ".circleci/build-requirements.txt" }}
```

Comment:
What is that related?

Reply:
Some changes made by Gal Rabin to test his fix. Reverting this